engine/src/engine.lcb

```diff
@@ -360,17 +365,34 @@ Example:
 		execute script "import snapshot from rect (the rect of widget " & tVar & ") of widget " & tVar
 	end handler
 
+Example:
+   public handler SnapshotMeAtSize(in pWidth as Integer, in pHeight as Integer) returns nothing
+		variable tVar as String
+		get property "number" of my script object
+		put the result formatted as string into tVar
+
+		// Create a snapshot from the rect of the widget
+		execute script "import snapshot from me at size at size param(1),param(2)" \
+            with [ pWidth, pHeight ]
+	end handler
+
 Description:
-Executes the given piece of LiveCode script in the context of the current script
-object.
+Executes the given piece of LiveCode script in the context of the target script
+object, or the current script object if none is specified. The list of arguments
+is accessible from the script fragment using the paramCount() and param()
+functions.
```

Comment:
Is it possible to explain more clearly how "the current script object" is determined? Is it the same as `my script object`? If so, maybe a cross-reference might make sense

Could you please put `paramCount()` and `param()` in inline code markup?